.github/workflows/remove-needs-triage.yml

```diff
@@ -0,0 +1,13 @@
+name: Remove needs-triage label if it is assigned
+
+on:
+  issues:
+    types: [assigned]
```

Comment:
The core set of rules also remove this if another label (not "needs-triage") is applied

Reply:
... and skip it if any labels are already on the issue (including "needs-triage")

Reply:
I am still investigating how to check "if it is labeled" in action. :)